api/src/main/resources/descriptions/io.strimzi.api.kafka.model.authentication.KafkaClientAuthenticationOAuth.adoc

```diff
@@ -0,0 +1,93 @@
+To use OAuth client authentication, set the `type` property to the value `oauth`.
+OAuth authentication can be configured in several different ways.
+
+.Client ID and secret
+
+You can configure the address of your OAuth server in the `tokenEndpointUri` property together with the OAuth client ID and client secret.
+The OAuth client will connect to the OAuth server, authenticate using the client ID and secret and get and access token which it will use to authenticate with the Kafka broker.
```

Comment:
```suggestion
The OAuth client will connect to the OAuth server, authenticate using the client ID and secret and get an access token which it will use to authenticate with the Kafka broker.
```